action-server/tests/action_helper.py

```diff
@@ -51,9 +51,9 @@ def create_tracker(
             last_action,
         )
 
-    def run_action(self, tracker: Tracker):
+    def run_action(self, tracker: Tracker, domain: Dict[Text, Any] = {}):
```

Comment:
You have to use None for the default domain parameter value, then set it to an empty dict in the function itself. With your current parameter definition, the empty dict is created only once, not each time the function is executed.